plugins/generator-1.19.4/forge-1.19.4/mappings/gamerules.yaml

```diff
@@ -0,0 +1,48 @@
+_default: null
+_mcreator_prefix: "CUSTOM:"
+_mcreator_map_template: "@JavaModNameGameRules.@REGISTRYNAME"
+ANNOUNCE_ADVANCEMENTS: GameRules.RULE_ANNOUNCE_ADVANCEMENTS
```

Comment:
Please order this file the way GameRules are in GameRules.java file, it makes reviewing mappings easier.

No need to do this with datalist file

Reply:
Done!

Reply:
Thanks!